tests/test_browser_cdp_capture.py

```
@@ -0,0 +1,130 @@
+import json
+import unittest
+
+from keyleak.browser_scanner import (
+    CDP_MAX_BODY_BYTES,
+    _CdpNetworkCapture,
+    _decode_cdp_body,
+    _is_text_mime,
+)
+
+
+SECRET = "sk-proj-ABCDEF1234567890SECRETVALUE0987654321"
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "SECRET"

(S105)